src/Ledger/Introduction.lagda

```diff
@@ -180,7 +180,8 @@ irrelevant. Additionally, all sets in this specification are finite.
 We use this set theory to define maps as seen below, which are used in
 many places. We usually think of maps as partial functions
 (i.e. functions not defined everywhere), but importantly they are not
-Agda functions.
+Agda functions. We denote the powerset operation by \PowerSet, which
+we use here to form a type of sets with elements in a given type.
```

Comment:
The phrase "type of sets with elements in a give type" sounds a bit odd... like a wordy way of describing the type of sets.  I mean, aren't all sets collections elements of a given type?  Are we using \PowerSet here in the usual (set of all subsets) sense?

Reply:
In classical set theory there are no types, so this sentence would be nonsensical. `{1, ℕ → ℕ}` is a perfectly fine set. And our set theory could be one where this is allowed, but it isn't.